### PR TITLE
use stringbuilder instead of string

### PR DESCRIPTION
**Pull request by cuongtranba**

### CODE REVIEW

**Comment by herrylove72** [comment]
please code review , thanks

File: VerbalExpressions/VerbalExpressions.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@
  */
 
 using System;
+using System.Diagnostics;
```
Comment (jehna) [inline VerbalExpressions/VerbalExpressions.cs]: I believe this import isn't used in the file?

Comment (herrylove72) [inline VerbalExpressions/VerbalExpressions.cs]: Yes, sorry for my mistake

**Comment by abatishchev** [comment]
Any chance to review, please?

**Pull request merged**